### ISSUE TITLE
Task fails with Gradle 7.0.2

### ISSUE THREAD

**Issue by guillaume-tgl** [twilio/apkscale#9]
> Before filing an issue please check the issue is not already addressed by the following:
>  * [Github Issues](https://github.com/twilio/apkscale/issues)
>  * [Changelog](https://github.com/twilio/apkscale/blob/master/CHANGELOG.md)

**Describe the bug**
When running the task with Gradle 7.0.2, an error is thrown: 

```
A problem was found with the configuration of task ':myApp:measureSize' (type 'MeasureAndroidLibrarySizeTask').
  - Type 'com.twilio.apkscale.tasks.MeasureAndroidLibrarySizeTask' property 'ndkVersion$apkscale' is missing an input or output annotation.
    
    Reason: A property without annotation isn't considered during up-to-date checking.
    
    Possible solutions:
      1. Add an input or output annotation.
      2. Mark it as @Internal.
    
    Please refer to https://docs.gradle.org/7.0.2/userguide/validation_problems.html#missing_annotation for more details about this problem.
```

**To Reproduce**
1. run `gradle myApp:measureSize`

**Version**
- Version: 0.1.2
- Gradle: 7.0.2
- AGP: 7.0.2

**Comment by aaalaniz** [comment]
Hey @guillaume-tgl 

This issue will be resolved once #8 has been merged and released as `0.1.3`. I have published `0.1.3-SNAPSHOT` that should resolve this issue #9 and #5 

You can pull in the `0.1.3-SNAPSHOT` by adding the following to your `build.gradle`.

```groovy
        maven {
            url 'https://oss.sonatype.org/content/repositories/snapshots/'
            mavenContent {
                snapshotsOnly()
            }
        }
        
        classpath "com.twilio:apkscale:0.1.3-SNAPSHOT"
```

Thanks!

**Comment by guillaume-tgl** [comment]
Excellent! I'll try the snapshot version and let you know if I have any issue.

**Comment by aaalaniz** [comment]
Fixed in 0.1.3
